games-action/heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
+HOMEPAGE="http://hota.sourceforge.net/"
+COPYRIGHT="2004 Gil Megidish"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="8f1f525fcf50c4610302e870c85625c8ea2f33eb"
+SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1f5d382c283a94a2c743746d7d3340671dfd965c4394fe66a86fcccab759dfb2"
+SOURCE_FILENAME="heartofthealien-$portVersion~$srcGitRev.tar.gz"
+SOURCE_DIR="beos-heartofthealien-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	heartofthealien = $portVersion
+	cmd:heartofthealien = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libsdl_mixer
+	lib:libsmpeg
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	devel:libsdl_mixer
+	devel:libsmpeg
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	cmd:touch
+	"
+
+BUILD()
+{
+	make -C src
+}
+
+INSTALL()
+{
+	install -d $appsDir/heartofthealien $docDir
+	install -t $appsDir/heartofthealien src/alien
+	install -t $appsDir/heartofthealien "Heart Of The Alien (U)".*
+	install -t $appsDir/heartofthealien rungame.sh
+	install -t $docDir doc/*
+	addAppDeskbarSymlink $appsDir/heartofthealien/rungame.sh heartofthealien
```

Comment:
The link in the Deskbar would look nicer as "Heart\ of\ the\ Alien"

Reply:
Edited.